edge/pkg/devicetwin/dmiserver/server.go

```diff
@@ -83,6 +84,7 @@ func (s *server) MapperRegister(ctx context.Context, in *pb.MapperRegisterReques
 	s.dmiCache.MapperList[in.Mapper.Name] = in.Mapper
 	s.dmiCache.MapperMu.Unlock()
 
+	dmiclient.DMIClientsImp.CreateDMIClient(in.Mapper.Protocol, string(in.Mapper.Address))
```

Comment:
why add this line at here? we have already write this code at line 122

Reply:
It's a mistake, I have removed it